src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
+export const NewCommentForm = ({ postId, commentUpdate }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [title, setTitle] = useState('');
 
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
+  const handleClick = (event) => {
+    event.preventDefault();
+    commentUpdate(postId, name, email, title);
+    setName('');
+    setEmail('');
+    setTitle('');
+  };
 
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={handleClick}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          required
+          value={name}
+          onChange={({target}) => setName(target.value)}
```

Comment:
```suggestion
          onChange={({ target }) => setCommentPart(target.name, target.value)}
```
create method setCommentPart which should work like this:
```javascript
const setCommentPart = (key, value) => {
  setComment(prevComment => ({
    ...prevComment,
    [key]: value,  
  })
}
```